src/app/portal/features/site-header/site-header.component.html

```diff
@@ -1,9 +1,20 @@
-<mat-toolbar color="primary" class="flex justify-between">
+<mat-toolbar class="flex justify-between">
   <a routerLink="/">GHGA Logo</a>
 
   <mat-nav-list>
-    <a routerLink="/" class="mx-1">Home</a
-    ><a routerLink="/browse" class="mx-1">Browse</a></mat-nav-list
+    <a
+      [id]="route === '/' ? 'activePage' : ''"
+      mat-button
+      routerLink="/"
+      class="homeLink mx-1"
+      >Home</a
+    ><a
```

Comment:
I wonder whether this strange formatting of prettier is a bug or a feature?

Reply:
Might be feature, to show that there are more tags after the closing tag?